getzep/zep-js
# feat/users

- users and tests
- formatting
- tests pass
- rename user emthods
- user class docs
- user class docs
- add correct errors to Document and Collection class docs
- add correct errors to Memory class docs
- update memory example; fix bugs
- min server version
- ensure trailing or leading slashes don't mess server url
